images/capi/packer/ami/windows-2004.json

```
@@ -0,0 +1,10 @@
+{
+  "ami_filter_name": "Windows_Server-2004-English-Core-ContainersLatest-*",
+  "ami_filter_owners": "amazon",
+  "build_name": "windows-2004",
+  "distribution": "windows",
+  "distribution_version": "2004",
+  "root_device_name": "/dev/sda1",
+  "source_ami": "",
+  "windows_updates_kbs": "KB4588962 KB2267602"
+}
```

As we know this image will not work with a nested hypervisor is it worth setting that as default in 2004 images 
```suggestion
{
  "ami_filter_name": "Windows_Server-2004-English-Core-ContainersLatest-*",
  "ami_filter_owners": "amazon",
  "build_name": "windows-2004",
  "distribution": "windows",
  "distribution_version": "2004",
  "root_device_name": "/dev/sda1",
  "source_ami": "",
  "windows_updates_kbs": "KB4588962 KB2267602",
  "disable_hypervisor": "true"
}
```

---

I tested the configuration without disabling hypervisor and it surprisingly worked.

---

that actually doesn't surprise me I think its because hyperv now installs regardless using the dism method and will just silently fail if the virtualisation feature is missing from the CPU, the disable hypervisor is just there for cases when someone may leave the support on in environments that don't support it. glad we can now build 2004 images